### PR TITLE
Add UI Mock Assets

### PR DESCRIPTION
**Pull request by assassinsgreed**
We can discuss the mocks further.

### CODE REVIEW

**Comment by CrystalAQuick** [comment]
Conflicts should be resolved before merging

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+Some settings may be set based on the user activity, such as their event viewer preference. All settings are applied immediately and persist between sessions.  
+If the user chooses to clear all preferences or reset data, they will be asked to confirm.
+
+![Settings Activity](/app/src/main/ui-mocks/Settings.png "Settings Activity")
+
+### Events Screen
+The events screen displays all calendar events in a calendar or list view (pictured below). The user's view preference and last visited month (or first expanded, in list view) is stored as a preference and retrieved when the window is revisited.
+
+![Events Screen Calendar](/app/src/main/ui-mocks/Events\ (Calendar\ View).png "Events Activity - Calendar View") 
+![Events Screen List](/app/src/main/ui-mocks/Events\ (List\ View).png "Events Activity - List View")
```
Comment (CrystalAQuick) [inline README.md:51]: Not a huge fan of the UI design for the event list view. Looks too busy.

**Review by CrystalAQuick — COMMENTED** [review]
Looks good overall

File: README.md (hunk 1 above)
Comment (assassinsgreed) [reply README.md:51]: Agreed - let's revise and review as the feature matures.

**Review by CrystalAQuick — APPROVED** [review]

**Pull request merged**